server_admin/topics/authentication/webauthn.adoc

```diff
@@ -0,0 +1,197 @@
+[[_webauthn]]
+
+=== W3C Web Authentication (WebAuthn)
+
+{project_name} provides the limited support for https://www.w3.org/TR/webauthn/[W3C Web Authentication (WebAuthn)]. {project_name} works as WebAuthn's https://www.w3.org/TR/webauthn/#rp-operations[Relying Party (RP)].
+
+IMPORTANT: Please note that this WebAuthn support is still in development phase and not yet fully completed so that it is recommended to use this WebAuthn support in experimental use. Moreover, there is a possibility that this support's specification and user interfaces will be changed in the future.
+
+NOTE: Whether WebAuthn's operations succeeds depends on a user's WebAuthn supporting authenticator, browser and platform. If you use this WebAuthn support, please clarify to what extent those entities supports WebAuthn specification.
+
+The major restriction is as follows :
+
+- A user can register only one WebAuthn supporting Authenticator (called WebAuthn authenticator here)
+- On registering the WebAuthn authenticator, its https://www.w3.org/TR/webauthn/#attestation-statement[Attestation Statement] is not verified
+- Only Two-Factor Authentication (2FA) scenario is supported
+
+==== Setup
+
+The setup procedure of WebAuthn support for 2FA is the following :
+
+Enable User Registration::
+
+An administrator carries out the following operations on `Admin Console` :
+
+- open `Realm Settings -> Login` tab
+- set `User registration` switch ON and push `Save` button
+
+Enable Webauthn Authenticator Registration::
+
+An administrator carries out the following operations on `Admin Console` :
+
+- open `Authentication -> Required Actions` tab
+- push `Register` button
```

Comment:
General format suggestions:

- Being with a capital letter:  Push

- Add THE before the noun:   ...the 'Register'...

- Complete full sentences with a period:  button.

So "Push the 'Register ' button.

Reply:
Thank you. I'll fix this sentence as you suggested.